src/components/user/uc/users.uc.integration.test.js

```diff
@@ -23,14 +24,15 @@ describe('user use case', () => {
 	describe('getOrCreateTombstoneUserId', () => {
```

Comment:
We should probably add a use case to test for the issue described in the ticket.